test/unit/loadGetInitialProps.test.js

```diff
@@ -0,0 +1,52 @@
+/* global describe, it, expect */
+import { loadGetInitialProps } from '../../dist/lib/utils'
+
+describe('loadGetInitialProps', () => {
+  it('should throw if getInitialProps is defined as an instance method', () => {
+    class TestComponent {
+      getInitialProps () {}
+    }
+    const rejectPromise = loadGetInitialProps(TestComponent, {})
+    const error = new Error('"TestComponent.getInitialProps()" is defined as an instance method - visit https://err.sh/next.js/get-inital-props-as-an-instance-method for more information.')
+    return expect(rejectPromise).rejects.toEqual(error)
```

Comment:
Please note: There is a more [elegant way](https://jestjs.io/docs/en/expect.html#rejects) to test for rejected promises in jest, but this is not working with out [current jest version](https://github.com/facebook/jest/issues/3601).